mappings/net/minecraft/structrure/PineTree.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_853 net/minecraft/structrure/PineTree
```

Comment:
I should probably change this to PineTreeFeature but I can do that in part 2

Reply:
structrure < structure